xbatcher/features.py

```diff
@@ -1,7 +0,0 @@
-"""Functions for transforming xarray datasets into features that can
-be input to machine learning libraries."""
-
-
-def dataset_to_feature_dataframe(ds, coords_as_features=False):
-    df = ds.to_dataframe()
-    return df
```

Comment:
this was an untested relic of the past. @rabernat - I think we can remove this. Thoughts?

Reply:
🗑️